README.md

```diff
@@ -1,5 +1,9 @@
 # AWS Ubuntu Desktop VM Module
 
+## Diagram
+
+![image](https://github.com/InterweaveCloud/terraform_aws_ubuntu_desktop_vm/blob/TER-59-modify-readme-so-it-is-suitable-for-os-consumption/Architecture%20Diagram.png?raw=true)
```

Comment:
this path needs to be relative, you put a link to a github feature branch

Reply:
done